pip/req/req_file.py

```diff
@@ -193,6 +193,21 @@ def process_line(line, filename, line_number, finder=None, comes_from=None,
             finder.find_links.append(value)
 
 
+def break_args_options(line):
+    """Break up the line into an args and options string.  We only want to shlex
+    (and then optparse) the options, not the args.  args can contain markers
+    which are corrupted by shlex.
+    """
+    tokens = shlex.split(line)
```

Comment:
on 2nd thought, using shlex here is bogus, because we won't be shlex parsing the args later, so the split here should just be simple string split on ' ' I think.

i.e. supposing the user is using quoting, we won't be honoring it later, so we shouldn't be honoring it now.